src/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx

```diff
@@ -84,3 +84,23 @@ export const ContactInformationForm: React.FC<ContactInformationFormProps> = ({
     </Box>
   )
 }
+
+export const ContactInformationFormFragmentContainer = createFragmentContainer(
```

Comment:
Adding a fragment container to use the form in the SWA flow and the Request Price Estimate flow.